Recipes/Recipes.Application/Services/RecipesService.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Recipes.Application.DTOs.Recipes;
+using Recipes.Public;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Recipes.Application.Services;
+
+public class RecipesService : IRecipesService
```

Comment:
I wonder if we need this here. We have created our own client-sided RecipeService, that consumes the API in FrontEnd.Web project through HttpClient requests.

Reply:
I noticed you have resolved this conversation, but we still have the Recipes.Application project. Do we want to keep it?

Reply:
Sorry, I thought I removed the `Application` layer previously. We definitely don't need it anymore.